README.md

```diff
@@ -44,11 +44,24 @@ If you need to add features to the API, the best is to [`npm link`](https://docs
 Openfisca
 ---------
 
-```sh
+Openfisca relies on Python 2. You should [install it in a `virtualenv`](https://doc.openfisca.fr/for_developers.html#create-a-virtualenv) to prevent yourself from messing with your main python installation. You can either create the `virtualenv` yourself or rely on tools such as [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/) or [pew](https://github.com/berdario/pew):
+
+```bash
 cd mes-aides-ui
-npm run install-openfisca # ou pip install --upgrade -r openfisca/requirements.txt si vous utilisez un environnement virtuel
+virtualenv  --python=python2 .venv # To create your virtualenv in ./.venv (a hidden folder)
+source .venv/bin/activate # To activate your virtualenv
+pip install pip --upgrade # To make sure you're using pip latest version
+```
+
+`virtualenv  --python=python2 .venv` will fail if your python executable is not `python2`. If that command fails you can try `virtualenv  --python=python2.7 .venv`.
```

Comment:
I tried it and it said:

```
Running virtualenv with interpreter /usr/local/bin/python2
New python executable in /Users/sgmap/Dev/mes-aides-ui/.venv/bin/python2.7
```

I am thus not 100% sure that this will fail if my Python is not Python 2.